platform/feature-api/communication/remote-pod/tsconfig.json

```diff
@@ -9,9 +9,8 @@
     "rootDir": "./src",
     "module": "commonjs",
     "declaration": true,
-    "strict": true,
```

Comment:
Is this the "apparently useless directive" you mentioned? From a quick look at TypeScript documentation, it doesn't seem to be all that useless. But of course if we disable strict mode the code would still work :D

Reply:
There does not seem to make a difference, and it's not entirely clear why we require that. We'll put it back there when we positively know we need it...

Reply:
Also, the definition of ["strict" might change with version](https://www.typescriptlang.org/tsconfig#strict). Anyway, we probably want type safety, so no need to take it out...